datahub-frontend/app/auth/AuthModule.java

```diff
@@ -49,7 +61,17 @@ public AuthModule(final Environment environment, final com.typesafe.config.Confi
 
     @Override
     protected void configure() {
-        final PlayCookieSessionStore playCacheCookieStore = new PlayCookieSessionStore();
+        PlayCookieSessionStore playCacheCookieStore;
+        try {
+            final String aesKeyBase = _configs.getString(PAC4J_AES_KEY_BASE_CONF);
+            MessageDigest sha = MessageDigest.getInstance("SHA-1");
```

Comment:
It's not necessarily for security purposes that we use SHA-1. We are using is to generate a string from which we can take the first 16 bytes, since AES keys _must_ be a specific length, from a 'seed' or 'base' string